pkg/connector/user.go

```diff
@@ -170,6 +170,15 @@ func listUsers(ctx context.Context, client *okta.Client, token *pagination.Token
 	return oktaUsers, respCtx, nil
 }
 
+func listAllUsers(ctx context.Context, client *okta.Client, qp *query.Params) ([]*okta.User, error) {
+	oktaUsers, _, err := client.User.ListUsers(ctx, qp)
```

Comment:
ListUsers is paginated and only returns up to 200 results at a time, so this will only fetch the first 200 users.